types/catbox/index.d.ts

```diff
@@ -41,7 +41,7 @@ export interface EnginePrototype {
  * The Client object provides the following methods:
  * @see {@link https://github.com/hapijs/catbox#api}
  */
-export interface ClientApi {
+export class ClientApi {
```

Comment:
This still shouldn't be a class, as there is not a runtime object called `ClassApi` from what I can tell.